### ISSUE TITLE
Two buttons to the right of the download button on a mod page missing accessibility labels.

### ISSUE THREAD

**Issue by BrailleBennett** [modrinth/code#2283]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### What browsers are you seeing the problem on?

Firefox, Safari

### Describe the bug

On the mod page, for example [Jade's](https://modrinth.com/mod/jade) on the top bar next to the download button are two buttons that are missing an accessibility label. Voice Over's auto describer says they are a heart and a bookmark. I'm not sure what they truely are, but that is the location. Without accessibility labels,the screen reader just reads "button" which is very unclear for blind users.

### Steps to reproduce

1. Turn on a screen reader like voice over, NVDA, or windows narrator.
2. Go to a mod page, like [Jade's](https://modrinth.com/mod/jade).
3. Tab over to the top of the page where you can find the download button.
4. Tab right once
5. observe how the "heart" AI auto captioned button is just labeled at "button"
6. Tab right again
7. Observe how the button autocaptioned with AI as "bookmark" is just labeled as "button" in the screen reader.

### Expected behavior

The buttons would have labels like "favorite" or "follow"and "bookmark" or "save".

**Comment by BrailleBennett** [comment]
This does effect all resource types, not just mods.

**Comment by Prospector** [comment]
Thanks for reporting! A fix should be live on the website within a few minutes!
